src/biokbase/narrative/jobs/jobcomm.py

```diff
@@ -436,9 +420,9 @@ def _retry_jobs(self, req: JobRequest) -> None:
         self.send_comm_message(
             "new_job",
             {
-                "job_id_list": [
+                JOB_ID_LIST: [
                     result["retry"]["jobState"]["job_id"]
-                    for result in retry_results
+                    for result in retry_results.values()
```

Comment:
`retry_results` is now a dictionary, so update this code to fetch the new retry IDs